find-external/qpOASES/FindqpOASES.cmake

```diff
@@ -28,7 +28,7 @@ find_path(
   PATHS ${qpOASES_PREFIX} ${qpOASES_PREFIX}/include)
 find_library(
   qpOASES_LIBRARY
-  NAMES libqpOASES.so
+  NAMES libqpOASES.so libqpOASES.dylib qpOASES.lib
```

Comment:
I guess we should use a name without suffix or prefix.  Could you try with only `qpOASES` ?